lib/thrift/parser.ex

```diff
@@ -91,12 +91,13 @@ defmodule Thrift.Parser do
   #   - convert an atom to a binary and remove the "Elixir." we get from atoms
   #      like `Foo`
   #   - make sure values are valid module names (CamelCase)
-  defp namespace_string(nil), do: nil
+  defp namespace_string(nil), do: "Thrift.Generated"
+  defp namespace_string(""), do: nil
```

Comment:
So they would have to put an empty string to get the root namespace?

What if instead of doing this conversion here you handle the defaults when you parse the arguments in the corresponding mix tasks?  Then you could do something like

```
namespace = if Keyword.has_key?(config, :namespace) do
  Keyword.get(config, :namespace)
else
  "Thrift.Generated"
end
```

That would translate to having "Thrift.Generated" as a default and letting the user override it as `namespace: nil`.